app/forms/steps/appeal/dispute_type_form.rb

```diff
@@ -23,6 +24,7 @@ def choices
         [
           DisputeType::PENALTY,
           DisputeType::AMOUNT_OF_TAX,
+          DisputeType::AMOUNT_OF_TAX_OWED,
```

Comment:
I'm not sure how I feel about these – should we perhaps make them a bit more descriptive?